FONT.md

```diff
@@ -1,5 +1,7 @@
-## Fira Mono
+## Source Sans Pro
 
-https://github.com/mozilla/Fira
+[https://github.com/adobe-fonts/source-sans-pro]()
```

Comment:
The link should be in the format [adobe-fonts/source-sans-pro](https://github.com/adobe-fonts/source-sans-pro)